src/main/java/org/truffleruby/core/module/ModuleFields.java

```diff
@@ -87,7 +87,11 @@ public static void debugModuleChain(DynamicObject module) {
     private final ConcurrentMap<String, InternalMethod> methods = new ConcurrentHashMap<>();
     private final ConcurrentMap<String, RubyConstant> constants = new ConcurrentHashMap<>();
     private final ConcurrentMap<String, Object> classVariables = new ConcurrentHashMap<>();
+
+    // Maps a refined class to refinement module
     private final ConcurrentMap<DynamicObject, DynamicObject> refinements = new ConcurrentHashMap<>();
+
+    // Maps a refined class to activated refinement module
```

Comment:
What does `activated` mean? Is it caused by calling `using`?

Reply:
These are `activated` by calling refine. "Activated" means the module chain has been updated and the `activatedRefinements` are added to the lexical scope when yielding the refine body.

Normally, refinements are activated by calling `using` but the above refinements appear to be active using only refine for the purpose of adding them to the scope of the refine block.